README.md

```diff
@@ -11,58 +11,27 @@ User application for Agoric Inter Protocol--Vaults, BLD Boost, Liquidations, etc
 
 ### Setup
 
-1. Download and build the latest copy of `agoric-sdk`, including Agoric's Cosmic SwingSet.
+1. Run the agoric local chain
```

Comment:
How do we get the keplr seed or access to a faucet now? I think it's worth mentioning here.

Reply:
yes, i should add getting gov1 mnemonic from the a3p docker.

Reply:
@samsiegart I've been following this link to get seed phrase:
https://docs.agoric.com/guides/getting-started/#starting-the-dapp-smart-contract

And it involves setting up the `offer-up` dapp. Is there a better way to get the seed phrase, after starting the chain? What if I collect the `contract` folder from the `offer-up` dapp` and add it in `dapp-inter`? We can have a similar script in `package.json` to start the contract and get a seed. 

I'd greatly appreciate your guidance on this.